App.js

```diff
@@ -28,7 +28,9 @@ const App = () => {
   };
 
   const clearItems = () => {
-    setItems(() => { items.map(item => deleteItem(item.id)); });
+    if (items.length > 0) {
+      setItems(() => { items.map(item => deleteItem(item.id)); });
+    }
```

Comment:
Var litt kjapp der, er nok riktigere å sjekke `items` inni `setItems()`.

```suggestion
    setItems(() => { if (items.length > 0) items.map(item => deleteItem(item.id)); });
```

Reply:
Eller for å tilfredstille Prettier…
```suggestion
      setItems(() => {
        if (items.length > 0) {
          items.map(item => deleteItem(item.id));
        }
      });
```

Reply:
Tydeligvis ikke! Da kræsja den også. `if` utenfor `setItems` funker.

Reply:
Tydelig at jeg er rusten, må selvfølgelig også sjekke om `items` er satt. Dette funker i simulatoren.
```suggestion
    if (items && items.length > 0) {
      setItems(() => { items.map(item => deleteItem(item.id)); });
    }
```